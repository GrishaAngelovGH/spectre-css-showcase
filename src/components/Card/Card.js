const Card = () => (
  <div className='columns'>
    <div className='column col-4 col-mx-auto'>
      <div className='card bg-gray'>
        <div className='card-header'>
          <div className='card-title h5'>Google</div>
          <div className='card-subtitle text-gray'>Software and hardware</div>
        </div>
        <div className='card-body'>Organize the worldâ€™s information and make it universally accessible and useful.</div>
        <div className='card-image'>
          <img className='img-responsive' src='https://9to5mac.com/wp-content/uploads/sites/6/2014/08/yosemite.jpg?quality=82&strip=all' alt='OS X Yosemite' />
        </div>
        <div className='card-footer'>
          <a className='btn btn-primary mx-2' href='#cards'>Search</a>
          <a className='btn btn-primary' href='#cards'>Share</a>
        </div>
      </div>
    </div>
  </div>
)

export default Card