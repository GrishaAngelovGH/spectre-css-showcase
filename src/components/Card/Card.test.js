import { render } from '@testing-library/react'

import Card from './Card'

test('should render Card component', () => {
  const view = render(<Card />)

  expect(view).toMatchSnapshot()
})