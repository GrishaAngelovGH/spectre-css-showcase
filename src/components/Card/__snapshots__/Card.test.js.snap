// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Card component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="columns"
      >
        <div
          class="column col-4 col-mx-auto"
        >
          <div
            class="card bg-gray"
          >
            <div
              class="card-header"
            >
              <div
                class="card-title h5"
              >
                Google
              </div>
              <div
                class="card-subtitle text-gray"
              >
                Software and hardware
              </div>
            </div>
            <div
              class="card-body"
            >
              Organize the world’s information and make it universally accessible and useful.
            </div>
            <div
              class="card-image"
            >
              <img
                alt="OS X Yosemite"
                class="img-responsive"
                src="https://9to5mac.com/wp-content/uploads/sites/6/2014/08/yosemite.jpg?quality=82&strip=all"
              />
            </div>
            <div
              class="card-footer"
            >
              <a
                class="btn btn-primary mx-2"
                href="#cards"
              >
                Search
              </a>
              <a
                class="btn btn-primary"
                href="#cards"
              >
                Share
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="columns"
    >
      <div
        class="column col-4 col-mx-auto"
      >
        <div
          class="card bg-gray"
        >
          <div
            class="card-header"
          >
            <div
              class="card-title h5"
            >
              Google
            </div>
            <div
              class="card-subtitle text-gray"
            >
              Software and hardware
            </div>
          </div>
          <div
            class="card-body"
          >
            Organize the world’s information and make it universally accessible and useful.
          </div>
          <div
            class="card-image"
          >
            <img
              alt="OS X Yosemite"
              class="img-responsive"
              src="https://9to5mac.com/wp-content/uploads/sites/6/2014/08/yosemite.jpg?quality=82&strip=all"
            />
          </div>
          <div
            class="card-footer"
          >
            <a
              class="btn btn-primary mx-2"
              href="#cards"
            >
              Search
            </a>
            <a
              class="btn btn-primary"
              href="#cards"
            >
              Share
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
